README.md

```diff
@@ -51,7 +51,7 @@ you may rely on the previously cited libraries to achieve such features.
 
 ## Rust version requirements
 
-Since version 0.17.0, `ratatui` requires **rustc version 1.59.0 or greater**.
+Since version 0.21.0, `ratatui` requires **rustc version 1.65.0 or greater**.
```

Comment:
Are we sure that the next version will be `0.21.0`?

Reply:
Regardless, we can fix it afterwards if we settle on something else. It was a just question to get your input 🐻

Reply:
Here's what the cargo book has to say about it: https://doc.rust-lang.org/cargo/reference/semver.html#possibly-breaking-changing-the-minimum-version-of-rust-required

(tl;dr - minor version or major version bump is suggested, not just patch)

Reply:
I'd highly advocate for this to go to 0.21 regardless as there are a couple of small breaking changes in flight (e.g. the PR for MaskedText, which breaks anything where someone calls methods that take `Into<Text>' by with a `something.as_ref()` instead of a `something.as_str()`)

Reply:
Sounds reasonable.